features/workflow.feature

```diff
@@ -140,3 +140,15 @@ Feature: Workflow
         And the "SIMPLE" job is not triggered
         And the "MASTER-PR" job is not triggered
         And that "REGEX" build uses the same SHA as the "STAGING" build
+
+    @ignore @workflow-chainPR
+    Scenario: chainPR
+        Given an existing pipeline on "master" branch with the workflow jobs:
+            | job       | requires  |
+            | SIMPLE    | ~commit   |
```

Comment:
```suggestion
            | SIMPLE    | ~pr       |
```

Reply:
Thank you. Fixed with table misalignment.